docs/website/docs/developers/debugging/sanitizers.md

```diff
@@ -39,6 +39,58 @@ Enabling ASan in the IREE build is a simple matter of setting the
 cmake -DIREE_ENABLE_ASAN=ON ...
 ```
 
+#### Linking to the dynamic Asan runtime
+
+You may want to use ASan when using the python bindings.
+To achieve this you need to link to the ASan runtime dynamically instead of
+linking it statically into an executable.
+
+```shell
+cmake \
+  -DIREE_ENABLE_ASAN=ON \
+  -DCMAKE_EXE_LINKER_FLAGS=-shared-libasan \
+  -DCMAKE_SHARED_LINKER_FLAGS=-shared-libasan \
+  -DCMAKE_C_COMPILER=clang-12 \
+  -DCMAKE_CXX_COMPILER=clang++-12 \
+  ...
+```
+
+Then when running things the ASan runtime will have to be preloaded.
+```shell
+LD_PRELOAD=/usr/lib/llvm-12/lib/clang/12.0.0/lib/linux/libclang_rt.asan-x86_64.so \
+ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-12/bin/llvm-symbolizer \
+  python ...
+```
+
+If you want to run the IREE CUDA runtime driver it is likely you would need.
+```shell
+ASAN_OPTIONS="protect_shadow_gap=0"
+```
+
+Like this
+```shell
```

Comment:
the markdown linter wants blank lines around code blocks: https://github.com/openxla/iree/actions/runs/8088497554/job/22102652563?pr=16613#step:4:5
```suggestion
Like this

```shell
```